### PR TITLE
Add coverage test #1

### PR DESCRIPTION
**Pull request by nkbt**
Phew!

Here we go. Lean solution with only one tool and almost zero config.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,15 +4,14 @@
   "main": "build/index.js",
   "scripts": {
     "init": "rimraf .validate.json && rimraf .jshintrc",
-    "clean": "rimraf build && mkdir -p build/test",
+    "clean": "rimraf build",
```
Comment (nkbt) [inline package.json]: We don't need to build tests anymore. We can run `babel-node` directly

File: test/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 import test from 'tape';
-import hello from '../index';
+import hello from '../source/index';
```
Comment (nkbt) [inline test/index.js]: Because I don't want to build tests (to capture original content for coverage), import path should be updated to the correct one.

**Comment by nkbt** [comment]
And coverage is awesome

screen shot 2015-06-19 at 00 23 06

File: package.json (hunk 1 above)
Comment (nkbt) [inline package.json]: `babel-node` is shipped with babel.

**Comment by ericelliott** [comment]
Wow. So much better! \o/ :tada: :clap:

**Comment by nkbt** [comment]
It was quite a journey!

**Comment by ericelliott** [comment]
Totally worth it. =)

**Comment by nkbt** [comment]
No doubts, rewarding experience. Going to use some of the research results
in my day-to-day work and personal libs as well.



-- 
Sincerely yours, Nikita Butenko
Google Talk: nik@butenko.me
Skype: nikita.butenko
Mobile: +61 450 264 801

**Pull request merged**